### PR TITLE
setup for filling in menu items

### PR DESCRIPTION
**Pull request by eureka-cpu**
removed hard coded coins and setup the menu to accept the json file

### CODE REVIEW

**Comment by eureka-cpu** [comment]
Forgot to remove headless ui from just dependencies package.json

**Comment by adlerjohn** [comment]
Is this ready for review?

**Comment by eureka-cpu** [comment]
@headless-ui needs to be removed from the package.json, but yes

**Review by adlerjohn — COMMENTED** [review]
Looks reasonable, will defer to @luizstacio as the JS expert 😂

File: client/components/Assets.tsx [hunk 1]
```diff
@@ -1,19 +1,11 @@
...
 
-const coinList = [
-    {
-        name: 'ETH',
-        color: CoinETH,
-        amount: 0,
-        logo: ethLogo
-    }
-];
+const coinList = require('../lib/assets.json');
```
Comment (luizstacio) [inline client/components/Assets.tsx:8]: Prefer use of `import`

**Review by luizstacio — CHANGES_REQUESTED** [review]
One review, other than that is good! Nice work, looking forward to start refactoring

**Pull request merged**